mozilla-release/mobile/android/base/java/org/mozilla/gecko/onboarding/CliqzIntroPagerAdapter.java

```diff
@@ -148,6 +154,7 @@ public Object instantiateItem(final ViewGroup container, int position) {
             @Override
             public void onClick(View v) {
                 container.setVisibility(View.GONE);
+                mCliqzIntroClosedListener.enterFirstRunEditingMode();
```

Comment:
This can be `null`. Add an `if !=null` or, better, add a `@NonNull` annotation to the constructor signature